utilities/titandb/blob_gc_job_test.cc

```diff
@@ -214,14 +220,67 @@ class BlobGCJobTest : public testing::Test {
     }
     delete db_iter;
     ASSERT_FALSE(iter->Valid() || !iter->status().ok());
-    DestoyDB();
+    DestroyDB();
   }
 };
 
 TEST_F(BlobGCJobTest, DiscardEntry) { TestDiscardEntry(); }
 
 TEST_F(BlobGCJobTest, RunGC) { TestRunGC(); }
 
+TEST_F(BlobGCJobTest, PurgeBlobs) {
```

Comment:
Add one or two sentences of inline comment to describe the flow of the test would be better :)